07_Bootstrap/praktikum/src/index.html

```diff
@@ -0,0 +1,242 @@
+					<button
+						class="navbar-toggler"
+						type="button"
+						data-bs-toggle="collapse"
+						data-bs-target="#navbarNavDropdown"
+						aria-controls="navbarNavDropdown"
+						aria-expanded="false"
+						aria-label="Toggle navigation">
+						<span class="navbar-toggler-icon"></span>
+					</button>
+					<div class="collapse navbar-collapse" id="navbarNavDropdown">
+						<ul class="navbar-nav ms-auto">
+							<li class="nav-item px-1">
+								<a class="nav-link nav-name" href="#hero">Home</a>
+							</li>
+							<li class="nav-item px-1">
+								<a class="nav-link nav-name" href="#about">About</a>
+							</li>
+							<li class="nav-item px-1">
+								<a class="nav-link nav-name" href="#features">Features</a>
+							</li>
+							<li class="nav-item px-1">
+								<a class="nav-link nav-name" href="#">Screenshots</a>
+							</li>
+							<li class="nav-item px-1">
+								<a class="nav-link nav-name" href="#">Team</a>
+							</li>
+							<li class="nav-item px-1">
+								<a class="nav-link nav-name" href="#">Pricing</a>
+							</li>
+							<li class="nav-item dropdown px-1">
+								<a class="nav-link dropdown-toggle nav-name" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false"> Drop Down </a>
+								<ul class="dropdown-menu">
+									<li><a class="dropdown-item" href="#">Action</a></li>
+									<li><a class="dropdown-item" href="#">Another action</a></li>
+									<li><a class="dropdown-item" href="#">Something else here</a></li>
+								</ul>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link nav-name" href="#">Contact</a>
+							</li>
+						</ul>
+					</div>
+				</div>
+			</nav>
+		</header>
+		<!-- Navbar End -->
+
+		<!-- Main Goes Here -->
+		<main class="main-body">
+			<!-- Hero Section Start -->
+			<section id="hero" class="hero">
+				<div class="container mx-auto">
+					<div class="text-center mb-5 text-black-50">
+						<h2 class="hero-title text-wrap fw-lighter display-5">Simple Systeme Fordiscount</h2>
+						<p class="d-block mt-3 lead fs-6 fw-normal">Integer Cursus Bibendum Augue Ac Cursus</p>
+					</div>
+				</div>
+			</section>
+			<!-- Hero Section End -->
+
+			<!-- Features Section Start -->
+			<section id="features" class="features">
+				<div class="container">
+					<div class="features-content row text-center g-5 py-5 row-cols-1 row-cols-lg-3 mx-auto">
+						<div class="col mb-3">
+							<img class="img-feature" src="/assets/img/svg/cloud.svg" alt="Cloud Image SVG" width="60" />
+							<h5 class="title-feature-1 text-uppercase fw-bold pt-4 pb-3 text-dark">Introducing Whatsapp</h5>
+							<p class="description-feature-1 pb-2 fw-normal text-black-50">Lorem ipsum is simply dummy test of the printing and typesetting industry</p>
+							<a class="read-more text-decoration-none fw-semibold" href="#">Read More</a>
+						</div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)

Reply:
Baik mas, terima kasih mas untuk referensinya